src/pages/_app.tsx

```diff
@@ -44,6 +44,16 @@ export default function App({ Component, pageProps }: AppProps) {
         />
         <meta name="twitter:site" content="@nypl" />
         <meta name="twitter:creator" content="@nypl" />
+        <meta
+          name="google-site-verification"
+          content="_shbOK1otHA_eFNFgJsOwITZrWQwRg4wr8nmrJPNVDM"
+        />
+        <meta name="csrf-param" content="authenticity_token" />
+        <meta
+          name="csrf-token"
+          content="YGD90UPAnu6p7nAVRLc7kYIQ9hBGSuLSphWaMZdj/r/pv9PTpDmiEQwEpy0Cv6CN7emeN6vjQOprMKv/+q3WeA=="
+        />
```

Comment:
The CSRF token is dynamic and different every time a patron visits the site. So this should not be hard coded. At the moment for just the homepage, the app does not need a CSRF token.

This will be useful when validating a form or when a user is signed in and needs to perform authenticated actions, but right now this is ok to leave out. 

It'd be good to know how this token is being generated and how it's used for future reference.

Reply:
Also remove the csrf-param tag?

Reply:
Yes, thanks.